python/tests/calculators.py

```diff
@@ -5,10 +5,35 @@
 from rascaline import SortedDistances
 from rascaline.calculator import DummyCalculator
 
+from rascaline.clib import _get_library, _set_default_logging_callback
+import ctypes
+
 from test_systems import TestSystem
 
 
+
 class TestDummyCalculator(unittest.TestCase):
+    def test_callback(self):
+        recorded_levels = []
+        recorded_messages = []
+        def test_callback(level, message):
+            recorded_levels.append(level)
+            recorded_messages.append(message)
+        test_callback_ctype = ctypes.CFUNCTYPE(None, ctypes.c_int, ctypes.c_char_p)(test_callback)
+        _get_library().rascal_set_logging_callback(test_callback_ctype)
+
+        system = TestSystem()
+        calculator = DummyCalculator(cutoff=3.2, delta=2, name="", gradients=True)
+        descriptor = calculator.compute(system)
+
+        print("recorded_messages", recorded_messages)
+        targeted_message = b"INFO:rascaline::calculators::dummy_calculator -- Computation of DummyCalculator is invoked."
```

Comment:
This should not have to be a byte string, but a standard python string instead

Reply:
changed